### PR TITLE
Use next ipfs-log

### PR DESCRIPTION
**Pull request by haadcode**
This PR adds changes needed to support the new version of ipfs-log with loading optimizations.

### CODE REVIEW

File: test/replicate-and-load.test.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const ipfsPath2 = './orbitdb/tests/replicate-and-load/2/ipfs'
 
 Object.keys(testAPIs).forEach(API => {
   describe(`orbit-db - Replicate and Load (${API})`, function() {
-    this.timeout(config.timeout)
+    this.timeout(config.timeout * 2)
```
Comment (aphelionz) [inline test/replicate-and-load.test.js]: What factors are responsible for the `* 2`?

**Comment by phillmac** [comment]
> I'm 100% guilty of this myself, but I'm thinking there's _got_ to be a better way to store these IPFS fixture blocks in git. 😆 I'm thinking I might try my hand at a zip file or tar.gz that is uncompressed via `make build.` What do you think?

what about git lfs? https://git-lfs.github.com/

File: test/replicate-and-load.test.js (hunk 1 above)
Comment (haadcode) [reply test/replicate-and-load.test.js]: I believe CI was taking too long for the browser tests and it timed out.

**Comment by haadcode** [comment]
This PR depends on:
- https://github.com/orbitdb/ipfs-log/pull/275
- https://github.com/orbitdb/orbit-db-identity-provider/pull/53
- new release of https://github.com/orbitdb/orbit-db-store
- new release of https://github.com/orbitdb/orbit-db-io

**Pull request merged**